prow/jobs/images/Dockerfile.integration

```diff
@@ -0,0 +1,40 @@
+# Dockerfile for integration testing
+FROM prow/test as base
```

Comment:
We don't need to mention the full image path here?
i.e. "public.ecr..../aws-controllers-k8s/prow/test"

Reply:
I don't actually push this image to the ECR public repo - it's just the first of a multi-stage build. The `build-images.sh` script builds them in order.